### PR TITLE
Grammar and spelling fixes in eip-725

### PR DESCRIPTION
**Pull request by TalAter**
Minor fixes to EIP 725

### CODE REVIEW

File: EIPS/eip-725.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ function getKey(bytes32 _key) constant returns(uint256[] purposes, uint256 keyTy
 
 #### keyHasPurpose
 
-Returns the `TRUE` if a key has is present and has the given purpose. If key is not present it returns `FALSE`.
+Returns the `TRUE` if a key is present and has the given purpose. If key is not present it returns `FALSE`.
```
Comment (androolloyd) [inline EIPS/eip-725.md]: Returns the 'TRUE' is also poor grammar.

**Comment by Arachnid** [comment]
Ping @frozeman

**Comment by TalAter** [comment]
@Arachnid 's pings have the lowest latency 👍

**Comment by axic** [comment]
@TalAter this is in conflict now, can you rebase and resolve the conflict?

**Pull request merged**